test/apiv2/23-containersArchive.at

```diff
@@ -0,0 +1,73 @@
+# -*- sh -*-
+#
+# test more container-related endpoints
+#
+
+red='\e[31m'
+nc='\e[0m'
+
+podman pull $IMAGE &>/dev/null
+
+# Ensure clean slate
+podman rm -a -f &>/dev/null
+
+CTR="ArchiveTestingCtr"
+
+TMPD=$(mktemp -d)
```

Comment:
Maybe I could use `${WORKSPACE}` instead of this? I don't know.

Reply:
LGTM